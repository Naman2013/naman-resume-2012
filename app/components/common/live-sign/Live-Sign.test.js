import React from 'react';
import renderer from 'react-test-renderer';
import LiveSign from './live-sign';

// const TEST_DATA = {
//   targetDivID: 'advertisement-target',
//   adURL: 'slooh.com',
//   adWidth: 300,
//   adHeight: 250,
// };

it('renders correctly', () => {
  const liveSign = renderer.create(
    <LiveSign />,
  ).toJSON();

  expect(liveSign).toMatchSnapshot();
});
