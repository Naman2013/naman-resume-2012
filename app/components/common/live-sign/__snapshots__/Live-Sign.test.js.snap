// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="signContainer"
  data-jsx={298085615}
>
  <div
    className="circle circle_blink"
    data-jsx={298085615}
  />
  <div
    className="liveSignContainer"
    data-jsx={298085615}
  >
    <div
      className="liveSign"
      data-jsx={298085615}
    >
      <div
        className=""
        data-jsx={298085615}
      >
        LIVE
      </div>
    </div>
  </div>
</div>
`;
